src/CIFAR100/README.md

```diff
@@ -0,0 +1,85 @@
+# CIFAR-100
+
+Description from the [original
+website](https://www.cs.toronto.edu/~kriz/cifar.html)
+
+> The CIFAR-10 and CIFAR-100 are labeled subsets of the
+> [80 million tiny images](http://people.csail.mit.edu/torralba/tinyimages/)
+> dataset. They were collected by Alex Krizhevsky, Vinod Nair,
+> and Geoffrey Hinton.
+>
+> This dataset is just like the CIFAR-10, except it has 100
+> classes containing 600 images each. There are 500 training
+> images and 100 testing images per class. The 100 classes in the
+> CIFAR-100 are grouped into 20 superclasses. Each image comes
+> with a "fine" label (the class to which it belongs) and a
+> "coarse" label (the superclass to which it belongs).
+
+## Usage
+
+This sub-module provides a programmatic interface to download,
+load, and work with the CIFAR-100 dataset.
+
+```julia
+using MLDatasets
+
+# download dataset
+CIFAR100.download()
```

Comment:
I'ld delete this line from the example, since it will be downloaded automatically, and that automatic download is a feature you want to showoff. No?

Reply:
I suppose. Really I do want to get rid of the sub-readmes anyway and create a nice documenter site